catroid/src/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -575,6 +621,7 @@ public void dispose() {
 		font.dispose();
 		axes.dispose();
 		disposeTextures();
+		disposeClonedSprites();
```

Comment:
Is this called when you leave the stage ? Just curious, because the cloned Sprites need to be deleted when we return from stage !?